NuGet.Config

```diff
@@ -4,5 +4,6 @@
     <clear />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="myget-legacy" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This had to be added because many of the newer packages referenced by VS 2022 SDK wasn't found in dotnet-public feed.